ui/app/src/components/ItemDisplay/utils.tsx

```diff
@@ -48,13 +48,13 @@ export function getItemStateText(stateMap: ItemStateMap, values?: Record<string,
       stateMap.submitted ? (
         <FormattedMessage id="itemState.submittedToStaging" defaultMessage="Submitted to staging" />
       ) : (
-        <FormattedMessage id="itemState.scheduledForStaging" defaultMessage="Scheduled for staging" />
+        <FormattedMessage id="itemState.scheduledToStaging" defaultMessage="Scheduled to staging" />
```

Comment:
You must update this "key" on all localization json files. All changes in keys or defaultMessage must be replicated to the json files if they are there already.